src/main/java/frc/robot/Constants.java

```diff
@@ -1,10 +1,14 @@
-package frc.robot;
-
-
-public final class Constants {
-
-    public static class ExampleSubsystem {
-        public static final double POWER = 0.5; // [%]
-    }
-
-}
+package frc.robot;
+
+import edu.wpi.first.wpilibj.util.Color;
+
+public class Constants {
+
+    public static class Conveyor {
+        public static final double POWER = 0.5; // [%]
+        public static final int MAX_CARGO_AMOUNT = 2;
+        public static final Color BLUE = new Color(0, 0, 0);
+        public static final Color RED = new Color(0, 0, 0);
+        public static final Color NONE = new Color(0, 0, 0);

```

Comment:
Maybe `NONE`'s color should be the color of the surface the color sensor sees when there is no Cargo facing it.